src/policy/fees.h

```diff
@@ -225,19 +225,19 @@ class CBlockPolicyEstimator
     void processBlock(unsigned int nBlockHeight,
                       std::vector<CTxMemPoolEntry>& entries, bool fCurrentEstimate);
 
-    /** Process a transaction confirmed in a block*/
+    /** Process a transaction confirmed in a block */
```

Comment:
If we added the CommentPragmas spacing then this would be taken care of as well I believe.